devel/lab/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup        github.com/zaquestion/lab 0.14.0 v
+categories          devel
+platforms           darwin
+license             public-domain
+maintainers
```

Comment:
This line cannot be blank. Are you willing to maintain this port? If not, then `nomaintainer` should be added.

Reply:
> For addresses in other domains, e.g., <account@example.org>, use the convention example.org:account to specify the address.

@pmetzger We don't recommend putting email address as is into Portfiles, see https://guide.macports.org/#reference.keywords.maintainers.

Reply:
```suggestion
-maintainers
+maintainers         nixgeex.org:c.wrinn openmaintainer
```
like this?

Reply:
@l2dy Yah, my mistake. I was tired when I wrote that.

Reply:
It doesn't seem to like my email.